File: src/main/java/ru/umeta/libraryintegration/application/BootApplication.java
package ru.umeta.libraryintegration.application;

/**
 * Created by ctash on 29.01.16.
 */

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@EnableAutoConfiguration
public class BootApplication {
    public static void main(String[] args) {
        SpringApplication.run(BootApplication.class, args);
    }
}

File: src/main/kotlin/ru/umeta/libraryintegration/BootApplication.kt
package ru.umeta.libraryintegration

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication

/**
 * Created by ctash on 02.02.16.
 */
@EnableAutoConfiguration
@SpringBootApplication
open class BootApplication {
    companion object {
        @JvmStatic fun main(args: Array<String>) {
            SpringApplication.run(BootApplication::class.java, *args)
        }
    }
}